cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -214,6 +215,7 @@ macro(configure_sdk_darwin
   set(SWIFT_SDK_${prefix}_IMPORT_LIBRARY_SUFFIX "")
   set(SWIFT_SDK_${prefix}_STATIC_LINKING_SUPPORTED FALSE)
   set(SWIFT_SDK_${prefix}_STATIC_ONLY FALSE)
+  set(SWIFT_SDK_${prefix}_USE_BUILD_ID FALSE)
```

Comment:
Nitpick: is it worth to put a comment explaining that by default on Darwin platform a UUID is generated by default (possibly referencing a copy of the [ld man page](https://github.com/apple-oss-distributions/dyld/blob/d552c40cd1de105f0ec95008e0e0c0972de43456/doc/man/man1/ld.1))?

Reply:
Also, should we set this to `FALSE` for Windows as well in the `configure_sdk_windows` function below?

Reply:
Given that the Windows PR testing passed, it looks like it may be better to remove this line, and rely on the fact that a variable that is not defined is set to the empty string (and thus evaluates to false in an `if()` statement.)

Reply:
I think I prefer setting it explicitly, as it's what we've done for everything else. But yes, you're right, we don't strictly need to.